frontend/src/components/TodoListItem.tsx

```diff
@@ -1,9 +1,10 @@
+import { Todo } from "../models/todo.model";
 import { IconDelete } from "./Icons/IconDelete";
-export default function TodoListItem(props: any) {
+export default function TodoListItem(props: Todo) {
```

Comment:
You always create a new interface to define the props of a component. That makes it easier to add/remove props.

Create an interface named `TodoListItemProps` and put `todo: Todo` inside it. Also use destructuring here.